zgrab_schema.py

```diff
@@ -800,29 +805,23 @@
                     "compression":String(),
                 }),
             }),
-            "dh_key_exchange": SubRecord({
-                "params": SubRecord({
-                    "prime": SubRecord({
-                        "value":Binary(),
-                        "length":Integer()
-                    }),
-                    "generator": SubRecord({
-                        "value":Binary(),
-                        "length":Integer()
-                    }),
-                    "client_public": SubRecord({
-                        "value":Binary(),
-                        "length":Integer()
-                    }),
-                    "client_private": SubRecord({
-                        "value":Binary(),
-                        "length":Integer()
-                    }),
+            "key_exchange": SubRecord({
+                "ed25519sha256_params": SubRecord({
```

Comment:
Ah, that's because of the "RFC". https://git.libssh.org/projects/libssh.git/tree/doc/curve25519-sha256@libssh.org.txt

We should change the name to `curve25519_sha256_params`, to match the protocol name. And also, because this is key exchange, not signing, so it's `Curve25519` or `X25519` (a Montgomery curve), as opposed to `Ed25519` (an Edwards curve), which is used for signing.